gcc/config/riscv/riscv.md

```diff
@@ -381,6 +381,7 @@
 ;; vidiv       vector single-width integer divide instructions
 ;; viwmul      vector widening integer multiply instructions
 ;; vimuladd    vector single-width integer multiply-add instructions
+;; vsfmuladd   vector matrix integer multiply-add instructions
```

Comment:
Minor comment, please put this in the end of the comment, don't mixed with standard instruction